Package.swift

```diff
@@ -23,30 +23,24 @@ let swiftSettings: [SwiftSetting] = [
 
 let package = Package(
     name: "swift-openapi-runtime",
-    platforms: [
-        .macOS(.v10_15), .macCatalyst(.v13), .iOS(.v13), .tvOS(.v13), .watchOS(.v6), .visionOS(.v1)
-    ],
-    products: [
-        .library(
-            name: "OpenAPIRuntime",
-            targets: ["OpenAPIRuntime"]
-        )
-    ],
-    dependencies: [
-        .package(url: "https://github.com/apple/swift-http-types", from: "1.0.0"),
-    ],
+    platforms: [.macOS(.v10_15), .macCatalyst(.v13), .iOS(.v13), .tvOS(.v13), .watchOS(.v6), .visionOS(.v1)],
+    products: [.library(name: "OpenAPIRuntime", targets: ["OpenAPIRuntime"])],
+    dependencies: [.package(url: "https://github.com/apple/swift-http-types", from: "1.0.0")],
```

Comment:
Curious why this got reformatted now, even though the CI was green previously as well?

Reply:
Yeah, the other repos didn't get reformatted, so I wonder if we could undo it here: https://github.com/swift-server/swift-openapi-async-http-client/pull/43/files

Reply:
Sorry, that was accidental - I've reverted the formatting changes